packages/linear-genome-view/src/BasicTrack/components/ServerSideRenderedBlockContent.tsx

```diff
@@ -33,6 +33,12 @@ const useStyles = makeStyles(theme => ({
     color: theme.palette.error.main,
     overflowY: 'auto',
   },
+  blockLimitMessage: {
+    width: '100%',
+    background: theme.palette.action.disabledBackground,
+    padding: theme.spacing(2),
+    textAlign: 'center',
+  },
```

Comment:
I think this should be renamed, since there may be something in the future that wants to pass a React node for a message that has nothing to do with limits.

Reply:
renamed to blockReactNodeMessage